CHANGELOG.md

```diff
@@ -18,6 +18,16 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.24.0] - 2024-12-04
 
+### Release highlights
```

Comment:
Should we add this heading under "[Unreleased]" above too? I'm not sure. Pros: I won't forget to add it. Cons: Contributors might think they're supposed to decide whether to update it in their own PR (they shouldn't IMO).

Reply:
I think if I were a new contributor, I would just ignore the "release highlights" section if I were adding an entry to the changelog, not having the hubris to think that my change must be a release highlight 🤣

Reply:
Added it to the unreleased template.